cli/src/main/kotlin/commands/RequirementsCommand.kt

```diff
@@ -111,7 +111,6 @@ class RequirementsCommand : CliktCommand(help = "List the required command line
             println("${category}s:")
 
             tools.forEach { tool ->
```

Comment:
commit: Our internal setup still depends on that mechanism, so we'd need to fix this internally before this PR gets merged.

Reply:
Oh, really? You're bootstrapping ScanCode on every ORT pipeline run? If so, that's also an opportunity to save a few minutes (as I understand you're working on general performance improvements), esp. if you do `scancode --reindex-licenses` beforehand, like we do in the official `Dockerfile`.

Reply:
If nothing else is blocking the merge, then I'll leave the merge to you, so you can do it when time suits you.

Reply:
PS: After this PR I was also considering to upgrade everything to ScanCode 3.2.3 final.

Reply:
Thanks @sschuberth . I'm aware of the optimization potential, have spent quite a bit of work on it but ran into some really weird issues which is why we don't have it in place yet.

Reply:
So, does this mean we'll not be able to merge this PR any time soon?

Reply:
No it doesn't. It means that I'd have a hard time providing a timeline.